### PR TITLE
feature(theme) update new theme colors 

### PR DESCRIPTION
**Pull request by SorinC6**
- update exiting dark theme to match the new design requirements
task: https://emurgo.atlassian.net/browse/YOMO-1399

### CODE REVIEW

File: apps/wallet-mobile/src/YoroiApp.tsx [hunk 1]
```diff
@@ -49,6 +49,8 @@ export const YoroiApp = () => {
 
   const themeStorage = useThemeStorageMaker()
 
+  console.log('themeStorage READ', themeStorage.read())
```
Comment (banklesss) [inline apps/wallet-mobile/src/YoroiApp.tsx]:
```suggestion

```

File: apps/wallet-mobile/src/auth/PinInput/PinInput.tsx [hunk 2]
```diff
@@ -79,54 +80,62 @@ type PinPlaceholderProps = {
...
+      paddingHorizontal: 8,
+    },
+    pinCircle: {
+      width: 16,
+      height: 16,
+      borderRadius: 10,
+    },
+    pinCircleInactive: {
+      borderWidth: 2,
+      borderColor: '#3154CB',
```
Comment (banklesss) [inline apps/wallet-mobile/src/auth/PinInput/PinInput.tsx]: missing theme color?

File: apps/wallet-mobile/src/auth/PinInput/PinInput.tsx [hunk 3]
```diff
@@ -79,54 +80,62 @@ type PinPlaceholderProps = {
...
+    },
+    pinCircleActive: {
+      backgroundColor: '#3154CB',
```
Comment (banklesss) [inline apps/wallet-mobile/src/auth/PinInput/PinInput.tsx]: missing theme color?

File: apps/wallet-mobile/src/components/NumericKeyboard.tsx [hunk 4]
```diff
@@ -59,46 +62,59 @@ const Key = ({value, onPress}: KeyboardKeyProps) => {
...
+      flexDirection: 'row',
+    },
+    keyboardKey: {
+      flex: 1,
+      alignItems: 'center',
+      justifyContent: 'center',
+      borderTopWidth: 2 * StyleSheet.hairlineWidth,
+      borderLeftWidth: StyleSheet.hairlineWidth,
+      borderRightWidth: StyleSheet.hairlineWidth,
+      borderColor: '#B7B7B7',
```
Comment (banklesss) [inline apps/wallet-mobile/src/components/NumericKeyboard.tsx]: missing theme color?

File: apps/wallet-mobile/src/features/SetupWallet/useCases/DarkThemeAnnouncement/DarkThemeAnnouncement.tsx [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+      ...atoms.py_sm,
+    },
+    caption: {
+      color: color.el_gray_medium,
+      textAlign: 'center',
+      ...atoms.body_3_sm_regular,
+    },
+
+    toggle: {
+      paddingBottom: 44,
```
Comment (banklesss) [inline apps/wallet-mobile/src/features/SetupWallet/useCases/DarkThemeAnnouncement/DarkThemeAnnouncement.tsx]: theme padding 👀

File: apps/wallet-mobile/src/features/SetupWallet/useCases/DarkThemeAnnouncement/DarkThemeAnnouncement.tsx [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+const useStyles = () => {
+  const {color, atoms} = useTheme()
+  const styles = StyleSheet.create({
+    container: {
+      flex: 1,
+      backgroundColor: color.gray_cmin,
+    },
+    content: {
+      alignItems: 'center',
+      paddingTop: 60,
```
Comment (banklesss) [inline apps/wallet-mobile/src/features/SetupWallet/useCases/DarkThemeAnnouncement/DarkThemeAnnouncement.tsx]: theme padding 👀

File: apps/wallet-mobile/src/components/Icon/Backspace.tsx [hunk 7]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import Svg, {Path} from 'react-native-svg'
+
+type Props = {
+  size?: number
+  color?: string
+}
+
+export const Backspace = ({color = 'black'}: Props) => (
+  <Svg width="24" height="17" viewBox="0 0 24 17" fill="none">
```
Comment (michalszorad) [inline apps/wallet-mobile/src/components/Icon/Backspace.tsx]: size prop not used

**Review by stackchain — APPROVED** [review]

**Pull request merged**